legacy_promotions/db/migrate/20240622113334_move_adjustment_eligible_to_legacy_promotions.rb

```diff
@@ -0,0 +1,11 @@
+class MoveAdjustmentEligibleToLegacyPromotions < ActiveRecord::Migration[7.0]
+  def up
+    unless column_exists?(:spree_adjustments, :eligible)
+      add_column(:spree_adjustments, :eligible, :boolean, default: true)
```

Comment:
Now with Rails 7.0 as schema version you can make use of `if_not_exists: true` here

Reply:
Or I can just leave it like this, I don't think there's a big advantage to it.